.github/workflows/ci-on-push-s390x.yaml

```diff
@@ -0,0 +1,30 @@
+name: Kata Containers CI - s390x
```

Comment:
I think we can avoid adding this one, and just add the correct "runs-on" on the jobs we already have.

Reply:
yep, I will remove the platform. thanks!